docs/src/model_creation/dsl_basics.md

```diff
@@ -103,28 +103,28 @@ end
 ### [Bi-directional (or reversible) reactions](@id dsl_description_reaction_bundling_reversible)
 As is the case for the following two-state model:
 ```@example dsl_basics
-rn7 = @reaction_network begin
+rn_bidir = @reaction_network begin
     k1, X1 --> X2
     k2, X2 --> X1
 end
 ```
 it is common that reactions occur in both directions (so-called *bi-directional* reactions). Here, it is possible to bundle the reactions into a single line by using the `<-->` arrow. When we do this, the rate term must include two separate rates (one for each direction, these are enclosed by a `()` and separated by a `,`). I.e. the two-state model can be declared using:
 ```@example dsl_basics
-rn7 = @reaction_network begin
+rn_bidir = @reaction_network begin
```

Comment:
In some cases these names are nice to show that two models are identical. Now I just tried appending descriptors to their names instead.
(could also do a `rn_this == rn_that` check, but preferred to keep things very straightforward here)